cmd/node-disk-manager-webhook/main.go

```diff
@@ -32,6 +34,10 @@ type resourceCaches struct {
 	lvmVGCache        ctldiskv1.LVMVolumeGroupCache
 	storageClassCache ctlstoragev1.StorageClassCache
 	pvCache           ctlcorev1.PersistentVolumeCache
+	volumeCache       lhv1beta2.VolumeCache
+	backingImageCache lhv1beta2.BackingImageCache
+	lhNodeCache       lhv1beta2.NodeCache
```

Comment:
I think we should use a consistent naming scheme here. So either they should all be called `lhXXXX, e.g. `lhVolumeCache` or `lhBackingImageCache`, or the prefix `lh` should be removed from `lhNodeCache`.

```suggestion
	nodeCache       lhv1beta2.NodeCache
```